xml/vk.xml

```diff
@@ -2360,18 +2360,18 @@ typedef void <name>CAMetalLayer</name>;
         </type>
         <type category="struct" name="VkImportMemoryZirconHandleInfoFUCHSIA" structextends="VkMemoryAllocateInfo">
             <member values="VK_STRUCTURE_TYPE_IMPORT_MEMORY_ZIRCON_HANDLE_INFO_FUCHSIA"><type>VkStructureType</type> <name>sType</name></member>
-            <member>const <type>void</type>*                      <name>pNext</name></member>
+            <member optional="true">const <type>void</type>*                      <name>pNext</name></member>
             <member optional="true"><type>VkExternalMemoryHandleTypeFlagBits</type> <name>handleType</name></member>
             <member optional="true"><type>zx_handle_t</type>           <name>handle</name></member>
         </type>
         <type category="struct" name="VkMemoryZirconHandlePropertiesFUCHSIA" returnedonly="true">
             <member values="VK_STRUCTURE_TYPE_MEMORY_ZIRCON_HANDLE_PROPERTIES_FUCHSIA"><type>VkStructureType</type> <name>sType</name></member>
-            <member><type>void</type>*                            <name>pNext</name></member>
+            <member optional="true"><type>void</type>*                            <name>pNext</name></member>
```

Comment:
I think these are fine, but because the struct is tagged "returnedonly" I don't think this actually does anything? Should probably double check it's not outputting anything. It's probably harmless, but almost definitely pointless from the spec perspective.

Reply:
Ah, this is the downside of just replacing it all with automatic tools - didn't realize it didn't quite make sense in this situation. I think coming from the binding generator perspective this still makes sense, as you still need the `optional` flag to infer the correct type.

Reply:
Because it's returnedonly, the only VU it gets is 'pNext must: be NULL', whether the 'optional' is there or not.